lambda.tf

```
@@ -0,0 +1,157 @@
+######################################################################
+## https://docs.datadoghq.com/integrations/amazon_rds/?tab=enhanced
+
+locals {
+  lambda_enabled         = module.this.enabled && var.dd_api_key_source.resource != "" ? true : false
+  dd_api_key_resource    = var.dd_api_key_source.resource
+  dd_api_key_identifier  = var.dd_api_key_source.identifier
+  dd_api_key_arn         = local.dd_api_key_resource == "ssm" ? data.aws_ssm_parameter.api_key[0].arn : local.dd_api_key_identifier
+  dd_api_key_iam_actions = [lookup({ kms = "kms:Decrypt", asm = "secretsmanager:GetSecretValue", ssm = "ssm:GetParameters" }, local.dd_api_key_resource, "")]
+  url                    = format(var.dd_artifact_url, var.dd_artifact_filename)
+  filename               = format("%v-%v.zip", var.dd_artifact_filename, var.dd_git_ref)
+  dd_api_key_kms         = local.dd_api_key_resource == "kms" ? { DD_KMS_API_KEY = var.dd_api_key_kms_ciphertext_blob } : {}
+  dd_api_key_asm         = local.dd_api_key_resource == "asm" ? { DD_API_KEY_SECRET_ARN = local.dd_api_key_identifier } : {}
+  dd_api_key_ssm         = local.dd_api_key_resource == "ssm" ? { DD_API_KEY_SSM_NAME = local.dd_api_key_identifier } : {}
+  lambda_env             = merge(local.dd_api_key_kms, local.dd_api_key_asm, local.dd_api_key_ssm)
+}
+
+data "aws_ssm_parameter" "api_key" {
+  count = local.dd_api_key_resource == "ssm" ? 1 : 0
+  name  = local.dd_api_key_identifier
+}
+
+module "lambda_label" {
+  source     = "cloudposse/label/null"
+  version    = "0.24.1" # requires Terraform >= 0.13.0
+  context    = module.this.context
```

```suggestion
  attributes = ["lambda"]
  
  context    = module.this.context
```

we put context as the last field